e2e/gui/src/test/java/com/epam/pipeline/autotests/LimitMountsTest.java

```diff
@@ -0,0 +1,261 @@
+    private String storage4 = "limitMountsStorage" + Utils.randomSuffix();
+    private final String registry = C.DEFAULT_REGISTRY;
+    private final String testTool = C.TESTING_TOOL_NAME;
+    private final String group = C.DEFAULT_GROUP;
+    private final String mountDataStoragesTask = "MountDataStorages";
+
+    @BeforeClass(alwaysRun = true)
+    public void setPreferences() {
+        library()
+                .createStorage(storage1)
+                .createStorage(storage2)
+                .createStorage(storage3)
+                .clickOnCreateStorageButton()
+                .setStoragePath(storage4)
+                .clickSensitiveStorageCheckbox()
+                .ok();
+        givePermissionsToStorage(user, storage2);
+        givePermissionsToStorage(user, storage3);
+        givePermissionsToStorage(user, storage4);
+        cleanToolLimitMounts();
+        cleanUserLimitMounts();
+        logout();
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void deleteTestEntities() {
+        open(C.ROOT_ADDRESS);
+        logoutIfNeeded();
+        loginAs(admin);
+        library()
+                .removeStorageIfExists(storage1)
+                .removeStorageIfExists(storage2)
+                .removeStorageIfExists(storage3)
+                .removeStorageIfExists(storage4);
+        cleanToolLimitMounts();
+        cleanUserLimitMounts();
+    }
+
+    @Test(priority = 0)
+    @TestCase(value = {"2210_1"})
+    public void validateSelectDataStoragesToLimitMountsForm() {
+        loginAs(user);
+        navigationMenu()
+                .settings()
+                .switchToMyProfile()
+                .ensure(LIMIT_MOUNTS, text("All available non-sensitive storages"))
+                .assertDoNotMountStoragesIsNotChecked()
+                .limitMountsPerUser()
+                .ensureAll(disabled, SELECT_ALL, SELECT_ALL_NON_SENSITIVE)
+                .ensureAll(enabled, CLEAR_SELECTION, CANCEL, OK)
+                .storagesCountShouldBeGreaterThan(2)
+                .searchStorage(storage1)
+                .ensure(TABLE, not(text(storage1)))
+                .clearSelection()
+                .ensureAll(enabled, SELECT_ALL, SELECT_ALL_NON_SENSITIVE, OK)
+                .ensureNotVisible(CLEAR_SELECTION)
+                .searchStorage(storage4)
+                .ensure(TABLE, not(text(storage4)))
+                .cancel();
+    }
+
+    @Test(priority = 1)
+    @TestCase(value = {"2210_2"})
+    public void validateDoNotMountStoragesOptionInUserProfile() {
+        navigationMenu()
+                .settings()
+                .switchToMyProfile()
+                .doNotMountStoragesSelect(true);
+        tools()
+                .perform(registry, group, testTool, tool ->
+                    tool.run(this))
+                .showLog(getLastRunId())
+                .expandTab(PARAMETERS)
+                .ensure(configurationParameter("CP_CAP_LIMIT_MOUNTS", "None"), exist)
```

Comment:
"CP_CAP_LIMIT_MOUNTS" -> extract to constant, please

Reply:
Done